hangman.js

```diff
@@ -1 +1,28 @@
-console.log("welcome to hangman");
\ No newline at end of file
+var word = prompt("Choose your word").toLowerCase();
+var guesses = [];
+var incorrectGuessesLimit = 6;
+var letters = [];
+var countIncorrectGuesess = [];
+var incorrectGuesses = [];
+
+function convertWordToDashes(word){
+  var letters = word.split("");
```

Comment:
You already have a global variable letters on line 4 so you don't need to redefine it again here